editor/export/editor_export.cpp

```diff
@@ -83,16 +85,21 @@ void EditorExport::_save() {
 		config->set_value(section, "encryption_exclude_filters", preset->get_enc_ex_filter());
 		config->set_value(section, "encrypt_pck", preset->get_enc_pck());
 		config->set_value(section, "encrypt_directory", preset->get_enc_directory());
-		config->set_value(section, "script_encryption_key", preset->get_script_encryption_key());
+		credentials->set_value(section, "script_encryption_key", preset->get_script_encryption_key());
 
 		String option_section = "preset." + itos(i) + ".options";
 
 		for (const PropertyInfo &E : preset->get_properties()) {
-			config->set_value(option_section, E.name, preset->get(E.name));
+			if ((E.usage | PROPERTY_USAGE_SECRET) == E.usage) {
```

Comment:
I think this is equivalent and a more concise (and common in the codebase) way to do this check:
```suggestion
			if (E.usage & PROPERTY_USAGE_SECRET) {
```